searx/engines/mojeek.py

```diff
@@ -150,15 +160,13 @@ def fetch_traits(engine_traits: EngineTraits):
     import contextlib
 
     resp = network.get(base_url + "/preferences")
-    dom = html.fromstring(resp.text)
+    dom = html.fromstring(resp.text)  # type: ignore
 
     for code in eval_xpath_list(dom, f'//select[@name="{language_param}"]/option/@value'):
         with contextlib.suppress(UnknownLocaleError):
             Locale(code)
             engine_traits.languages[code] = code
 
     for code in eval_xpath_list(dom, f'//select[@name="{region_param}"]/option/@value'):
-        for locale in get_official_locales(code):
-            if locale.language in engine_traits.languages:
-                print('region:', code, locale.english_name)
-                engine_traits.regions[region_tag(locale)] = code
+        for locale in get_official_locales(code, engine_traits.languages, de_facto=False):
```

Comment:
Setting `de_facto=False` excludes regions such as `en-US`. So, I'm not sure if that is desired.

Reply:
> excludes regions such as `en-US`.

Oops .. thanks for pointing out, I fixed the issue --> https://github.com/searxng/searxng/compare/cc7dfbd2b0b1204687611dedc457caadf879408a..c98e83153c9e03b72bad9cd92404d97c201d8c3a